cobalt/tools/on_device_tests_gateway_client.py

```diff
@@ -344,20 +342,21 @@ def main() -> int:
   trigger_args.add_argument(
       '--job_timeout_sec',
       type=str,
-      default='1800',
-      help='Timeout in seconds for the job (default: 1800 seconds).',
+      default='2100',
```

Comment:
Do we need to increase it?

Reply:
Yes, the job timeout must be set higher than the test timeout. I think we should review these timeouts at some point. They can probably be lowered across the board.